splink/cluster_metrics.py

```diff
@@ -137,3 +138,27 @@ def _size_density_centralisation_sql(
     sqls.append(sql)
 
     return sqls
+
+
+@dataclass
+class GraphMetricsResults:
+    nodes: SplinkDataFrame
+    edges: SplinkDataFrame
+    clusters: SplinkDataFrame
+
+    def __repr__(self):
+        return """
+A data class of Splink dataframes containing metrics for nodes, edges and clusters.
+
+Access dataframes via attributes:
+`compute_graph_metrics.nodes` for node metrics,
+`compute_graph_metrics.edges` for edge metrics and
+`compute_graph_metrics.clusters` for cluster metrics
+
+or equivalently unpack like so:
+```node_metrics, edge_metrics, cluster_metrics = (
+    df_graph_metrics.nodes,
+    df_graph_metrics.edges,
+    df_graph_metrics.clusters,
+)```
```

Comment:
I _think_ that the suggestion was to be able to unpack directly, `node_metrics, edge_metrics, cluster_metrics = df_graph_metrics`, or more likely in practice:
```py
node_metrics, edge_metrics, cluster_metrics = linker.compute_graph_metrics(...)
```

so that you can immediately bypass the class itself if you want. Might want to check with Tom though if that's what he meant.

I'm personally not _massively_ fussed about this feature though, so happy to leave if it's going to be too fiddly

Reply:
I'm happy to leave for now and implement at a later date if the user need becomes apparent :)